backend/cinema/serializers.py

```diff
@@ -114,7 +114,10 @@ class TicketSerializer(serializers.ModelSerializer):
     def validate(self, attrs):
         data = super(TicketSerializer, self).validate(attrs=attrs)
         Ticket.validate_ticket(
-            attrs["row"], attrs["seat"], data["movie_session"].cinema_hall, ValidationError
```

Comment:
Consider using more descriptive variable names instead of 'attrs' and 'data' for better code readability. For example, 'attributes' and 'validated_data'.